sql-statements/sql-statement-kill.md

```diff
@@ -34,8 +34,8 @@ Query OK, 0 rows affected (0.00 sec)
 
 ## MySQL compatibility
 
-* By design, this statement is not compatible with MySQL by default. This helps prevent against a case of a connection being terminated on the wrong TiDB server, because it is common to place multiple TiDB servers behind a load balancer.
-* The `KILL TIDB` statement is a TiDB extension. If you are certain that the session you are attempting to kill is on the same TiDB server, set [`compatible-kill-query = true`](/tidb-configuration-file.md#compatible-kill-query) in your configuration file.
+* By design, `KILL` is not compatible with MySQL by default. This helps prevent against a case of a connection being terminated on the wrong TiDB server, because it is common to place multiple TiDB servers behind a load balancer. If you are certain that the session you are attempting to kill is on the same TiDB server, set [`compatible-kill-query = true`](/tidb-configuration-file.md#compatible-kill-query) in your configuration file.
```

Comment:
I don't think changing the configuration is a good way to handle a specific session's decision if it is safe to assume that the client will be connected to the same TiDB node, which is how I interpret the current text.

Maybe rephrase to something like:
Only change the configuration if you are certain that clients always will be connected to the same TiDB node, since the default mysql client on `ctrl-c` will open a new connection to kill the currently running command, and if there are proxies in between, the new connection might be routed to a different TiDB node possibly killing a different session.

Will `KILL [TIDB]` terminate the running query or terminate the connection? (see https://dev.mysql.com/doc/refman/8.0/en/kill.html `KILL [CONNECTION]` vs `KILL QUERY`)

Reply:
I agree with you.

Reply:
https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md

I think The Global Kill is the final solution. but it is experimental.